design/accepted/gc_plus/uncomitted-gc.md

```diff
@@ -79,19 +79,26 @@ objects added using this operation are never collected by GC.
 
 #### S3 Gateway CopyObject
 
-1. Copy object in the same branch will work the same - creating a new staging entry using the existing entry information.
-2. For objects that are not part of the branch, use the underlying adapter copy operation.
+When performing a shallow copy - track copied objects in ref-store.
+GC will read the copied objects information from the ref-store, and will add them to the list of uncommitted.
```

Comment:
The timing here is critical - it MUST read the new table after it iterated on the staging area of all branches.

Reply:
Explained in PrepareUncommittedForGC - added additional note to emphasize